src/log.py

```diff
@@ -0,0 +1,101 @@
+"""
+log
+
+By: Norma Chang
+"""
+import json
+import time
+
+
+class Log:
+    """
+    Log class.
+
+    Generates a log file that contains the data from the
+    properties of an instance of the leader/follower class.
+
+    Array of dictionaries is generated when the log file is loaded.
+    """
+    def __init__(self):
+        self._start_time = time.strftime('%b %d %Y %H:%M:%S', time.gmtime())
```

Comment:
this one was a bit difficult, I wanted to check the correct name of the vehicle before generating the file name with it so I didnt put it in init.

Reply:
Why is this more difficult than evaluating it later?

Reply:
What do you mean? I thought the request was to generate the filename in init? I attempted to do that but init shouldnt have if/else statements and I was doing that to check that the correct class was running before giving the file the correct class name otherwise it would just have the date. I thought it would be clearest to have the name of the class in the file name with the date.

Reply:
What's wrong with having an if/else statement in init?